libraries/AP_IOMCU/AP_IOMCU.cpp

```diff
@@ -420,6 +421,9 @@ bool AP_IOMCU::read_registers(uint8_t page, uint8_t offset, uint8_t count, uint1
         return false;
     }
     memcpy(regs, pkt.regs, count*2);
+    if (protocol_fail_count > 3) {
```

Comment:
Seems a bit tight given that we talked about shifting to a counter of 5 to be considered unhealthy.